phi/eval/eval.py

```diff
@@ -123,16 +131,14 @@ def get_accuracy_evaluator(self) -> Agent:
             response_model=AccuracyResult,
         )
 
-    def run(self, answer: Optional[Union[str, Callable]] = None) -> Optional[EvalResult]:
+    def run(self, question: str, expected_answer: str, answer: Optional[Union[str, Callable]] = None) -> Optional[EvalResult]:
```

Comment:
What is the use of answer here? I think its maybe a bad parameter name. Aren't we always supposed to get the answer from the agent?